statics/js/components/rule-detail.js

```diff
@@ -683,10 +683,11 @@ Vue.component('rule-detail', {
           return undefined
         },
         getTargets: function(node) {
+          var nodeid = node.id ? node.id : node.ID;
```

Comment:
could you please add here some mention // @todo, get rid of this line when we refactor default skydive layout.
Right now we need to keep in mind such differences between different node representation: backend returns data with ID property but on frontend level we use id property. This is a thing which should be optimized in future.